bundler/lib/bundler/cli/gem.rb

```diff
@@ -47,13 +47,16 @@ def run
       git_author_name = use_git ? `git config user.name`.chomp : ""
       git_username = use_git ? `git config github.user`.chomp : ""
       git_user_email = use_git ? `git config user.email`.chomp : ""
+      github_username = github_username(git_username)
 
-      github_username = if options[:github_username].nil?
-        git_username
-      elsif options[:github_username] == false
-        ""
+      if github_username.empty?
+        homepage_uri = "TODO: Put your gem's website or public repo URL here."
+        source_code_uri = "TODO: Put your gem's public repo URL here."
+        changelog_uri = "TODO: Put your gem's CHANGELOG.md URL here." if options["changelog"]
```

Comment:
I see you're guarding both `changelog_uri` assignments with `if options["changelog"]`. I guess you may as well not bother and set `config["changelog_uri"]` anyways, since it's still guarded by the same condition in the template. After all, the changelog uri config is correct even if the toggle to not display it is there.

Alternatively, if we keep the `if options["changelog"]` guards here, we could change the condition in the template to `if options["changelog_uri"]` instead of `if options["changelog"]`.

Alternatively, it's fine as is too :)